alpine/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# this is a slimmed-down version of https://git.alpinelinux.org/aports/tree/main/varnish/APKBUILD
+# Contributor: Pedro Filipe <xpecex@outlook.com>
+# Contributor: Leonardo Arena <rnalrd@alpinelinux.org>
+# Contributor: V.Krishn <vkrishn4@gmail.com>
+# Contributor: Natanael Copa <ncopa@alpinelinux.org>
+pkgname=varnish
+pkgver=@VERSION@
+pkgrel=1
+pkgdesc="High-performance HTTP accelerator"
+url="http://www.varnish-cache.org/"
+arch="all"
+license="BSD"
+options="!check" # tests pass, but ran before packaging
+depends="gcc libc-dev libgcc"
+depends_dev=
+makedepends="$depends_dev pcre-dev ncurses-dev libedit-dev py-docutils
+	linux-headers libunwind-dev python py-sphinx"
+install="$pkgname.pre-install"
+subpackages="$pkgname-dbg $pkgname-dev $pkgname-doc $pkgname-libs"
+pkgusers="varnish"
+pkggroups="varnish"
+source="$pkgname-$pkgver.tar.gz"
+
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	./configure \
+		--build="$CBUILD" \
+		--host="$CHOST" \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--infodir=/usr/share/info \
+		--localstatedir=/var/lib \
+		--without-jemalloc \
+		--with-unwind
+	make
+}
+
+check() {
+	cd "$builddir"
+
+	make -j12 check VERBOSE=1
+}
+
+package() {
+	cd "$builddir"
+
+	make DESTDIR="$pkgdir" install
+
+	local i; for i in $source; do
+		case $i in
+		*.initd) install -Dm755 "$srcdir/$i" \
+				"$pkgdir/etc/init.d/${i%.*}" ;;
+		*.confd) install -Dm644 "$srcdir/$i" \
+				"$pkgdir/etc/conf.d/${i%.*}" ;;
+		esac
+	done
+	install -d -o varnish -g varnish -m750 \
+		"$pkgdir"/var/cache/varnish \
+		"$pkgdir"/var/log/varnish \
+		"$pkgdir"/var/lib/varnish
+	install -d -o root -g varnish -m750 "$pkgdir"/etc/varnish
+}
+
+sha512sums="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX  varnish-$pkgver.tar.gz"
```

Comment:
wut?

Reply:
this file is a template to be filled, so we can't know the checksum for the file. We can get rid of the line altogether, but I think it's a nice marker that there's something to be done here

Reply:
I left it around, but also added a comment at the top explaining what needs to happen before using this file